docs/content/connectors/db2-cdc.md

```diff
@@ -0,0 +1,369 @@
+# Db2 CDC Connector
```

Comment:
`flink-cdc-connectors\docs\content\about.md` shoud be updated too and miss `quickstart`.